packages/cli/lib/sandbox-create.js

```diff
@@ -101,6 +101,22 @@ const buildSandbox = async ({
         })
       );
       logger.log('');
+    } else if (
+      isSpecifiedError(err, {
+        statusCode: 403,
+        category: 'BANNED',
+        subCategory: 'SandboxErrors.DEVELOPMENT_SANDBOX_ACCESS_NOT_ALLOWED',
```

Comment:
Added a gating error scenario for when users are not ungated for crm-developement-beta